content/sensu-go/6.3/observability-pipeline/observe-events/events.md

```diff
@@ -897,6 +897,10 @@ To show event details in the default [output format][18] (tabular):
 sensuctl event info entity-name check-name
 {{< /code >}}
 
+{{% notice note %}}
+**NOTE**: Metrics data points are not included in events retrieved with `sensuctl event info` &mdash; these events include text output rather than a set of metrics points.
```

Comment:
```suggestion
**NOTE**: Metrics data points are not included in events retrieved with `sensuctl event info` &mdash; these events include check output text rather than a set of metrics points.
```